store/tikv/2pc.go

```diff
@@ -378,10 +383,16 @@ func (c *twoPhaseCommitter) prewriteSingleBatch(bo *Backoffer, batch batchKeys)
 			log.Debugf("con:%d 2PC prewrite encounters lock: %v", c.connID, lock)
 			locks = append(locks, lock)
 		}
+		if len(locks) == 0 {
```

Comment:
Why do we need this check? Is this a bugfix?

Reply:
it's not a bugfix, `ResolveLocks` do the same check, check this here just to avoid useless `time.Now` call.

Reply:
Please add some comments to explain.

Reply:
review code `len(locks) == 0` always true if we can get here, so I decide remove this if.